src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css

```diff
@@ -22,17 +22,17 @@
   --sst-dark-color: var(--sst-sunpy-logo-gray);
   --sst-darker-color: #333333;
   --sst-darkest-color: #222222;
-  --sst-lightest-color: rgb(251, 251, 251);
-  --sst-lighter-color: rgb(231, 231, 231);
+  --sst-lightest-color: rgb(255, 255, 255);
+  --sst-lighter-color: rgb(245, 245, 245);
   --sst-light-color: rgb(206, 206, 206);
```

Comment:
These three were uniformly spaced in brightness. What motivated this change?

We should either change them all or none of them.

Reply:
The lack of contrast on the dark background. We want people to read grey text on a black background?

Reply:
I will undo the color changes we can go through them.